### ISSUE TITLE
i found a bug at rotation value.

### ISSUE THREAD

**Issue by Fogrunner** [hammerjs/hammer.js#859]
when i used turning gesture. sometime hammer gave me wrong rotation value.

it was absolutely opposite angle. for example, hammer should give me 45 degree value but it gave me 225. (Additional 180 degrees)

it always happened when i take off just one of my finger which touched first. i checked the other values
and found a different thing between event.pointers and event.srcEvent.touches.

srcEvent was correct. srcEvent has just one element. but pointers still has two elements. besides, 
pointers's values had swapped each other. p1 -> p2 and p2 -> p1! omg.

hence, hammer might send opposite value.

**Comment by runspired** [comment]
Related:
- https://github.com/hammerjs/hammer.js/issues/675
- https://github.com/hammerjs/hammer.js/issues/729

**Comment by runspired** [comment]
Also related:
#610
#791
#696

I believe this was fixed by https://github.com/hammerjs/hammer.js/commit/287720a6e5067e7f28be8b8b3b266d22905361c4

**Comment by runspired** [comment]
@arschmits we should discuss this, I have a demo with the new build here: http://codepen.io/runspired/pen/KdbzLb

TL;DR It jumps if you do not release in the inverse order you initially began touching.  While it appears the initial bug has been fixed, this is still an annoying behavior.  We should probably be "locking" rotation to 0 when the second pointer first lands, and "locking" rotation to it's final value when the second pointer is removed.

**Comment by runspired** [comment]
After digging in, it seems `force: 0;` on a changedPointer event does not result in Hammer removing the associated pointer.

**Comment by runspired** [comment]
Fixed by 287720a, an additional behavior improvement is under discussion.

**Comment by julian0852** [comment]
I have an issue with rotate. On first touch with two fingers the element immediately rotates through 180 deg. Even the rotate example at http://hammerjs.github.io/ does this strange behaviour. Any bug fix?

**Comment by zxmsdyz** [comment]
ios still have this problem not closed. i'm using 2.0.8 @runspired

**Comment by psivanov** [comment]
This is still hapenning. Specifically, when do the following, the last event has its pointers reversed.
1) touch with finger A
2) touch with finger B
3) move the fingers to rotate
4) untouch finger A
5) untouch finger B

**Comment by lizy0329** [comment]
still hapenning

**Comment by mayankkataria** [comment]
I'm still facing this issue.
